gneiss/plot/_blobtree.py

```diff
@@ -0,0 +1,218 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the GPLv3 License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+import ete3
+from ete3 import TreeStyle, AttrFace, NodeStyle
+from ete3.treeview import faces
+from PyQt4.QtGui import (QGraphicsPolygonItem,
+                         QPen, QColor, QBrush, QPolygonF)
+from PyQt4.QtCore import QPointF
+from ete3.treeview.faces import StaticItemFace, Face
+import pandas as pd
+
+
+class _DiamondItem(QGraphicsPolygonItem):
+    def __init__(self, width, height, label, color='#0000FF'):
+
+        self.pol = QPolygonF()
+
+        self.pol = QPolygonF()
+        self.pol.append(QPointF(width / 2.0, 0))
+        self.pol.append(QPointF(width, height / 2.0))
+        self.pol.append(QPointF(width / 2.0, height))
+        self.pol.append(QPointF(0, height / 2.0))
+        self.pol.append(QPointF(width / 2.0, 0))
+
+        self.label = label
+        QGraphicsPolygonItem.__init__(self, self.pol)
+
+        self.setBrush(QBrush(QColor(color)))
+        self.setPen(QPen(QColor(color)))
+
+    def paint(self, p, option, widget):
+        super(_DiamondItem, self).paint(p, option, widget)
+        ete3.treeview.faces._label_painter(self, p, option, widget)
+
+
+class CollapsedDiamondFace(StaticItemFace, Face):
+    """
+    Creates a collapsed node face object.
+
+    """
+    def __init__(self, width, height, label='', color='#0000FF'):
```

Comment:
@wasade right.  However, these classes are actually very different because of the classes that they inherit.  
`_DiamondItem` inherits from `QGraphicsPolygonItem` (from PyQT4) and needs to implement `paint` (since it is an abstract method).

`CollapsedDiamondFace` is an ETE object that inherits from both `StaticItemFace` and `Face`.  The main reason why I've have kept these classes separate was because that was the convention in [ETE](https://github.com/etetoolkit/ete/blob/master/ete3/treeview/faces.py).